utils/metrics/src/main/java/org/apache/edgent/metrics/MetricsSetup.java

```diff
@@ -85,6 +92,25 @@ public MetricsSetup startJMXReporter(String jmxDomainName) {
         reporter.start();
         return this;
     }
+
+    /**
+     * Starts the metric {@code CsvReporter}. If no MBeanServer was set, use the
+     * virtual machine's platform MBeanServer.
+     * 
+     * @param pathMetrics
```

Comment:
Sorry, what I was trying to ask does this javadoc need to be adjusted now?  Can you still supply "null" or a path to a non-existant file?